third_party/penumbra/examples/awning.cpp

```diff
@@ -40,7 +40,7 @@ int main(void) {
 
   unsigned wallId = pumbra.addSurface(wall);
   unsigned windowId = pumbra.addSurface(window);
-  unsigned awningId = pumbra.addSurface(awning);
+  // unsigned awningId = pumbra.addSurface(awning);
```

Comment:
Unused argument.  I believe @nealkruis will also be fixing this with a larger update to penumbra, but this gets it nice and clean right now.